packages/store-ui/src/atoms/List/stories/List.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import type { Story } from '@storybook/react'
+import React from 'react'
+
+import type { ComponentArgTypes } from '../../../typings/utils'
+import type { ListProps } from '../List'
+import Component from '../List'
+import mdx from './List.mdx'
+
+type ListStoryProps = ListProps & { options: string[] }
+
+const ListTemplate: Story<ListStoryProps> = ({ options, ...props }) => {
+  return (
+    <Component {...props}>
+      {options.map((value) => (
+        <li key={value}>{value}</li>
+      ))}
+    </Component>
+  )
+}
+
+export const List = ListTemplate.bind({})
```

Comment:
What do you think about create other examples using `as` property?

Reply:
Since we already have the `variant` as a control in the story I don't see the need for `or/unordered`, but a `description` list example makes sense